cypress/support/fragments/bulk-edit/bulk-edit-actions.js

```diff
@@ -249,7 +252,16 @@ export default {
       SelectionOption(including(location)).click(),
     ]);
   },
-
+  selectLocation(location, rowIndex) {
```

Comment:
see this method `replacePermanentLocation`

Reply:
I`ve seen that method, but in TC step by step I should do